coops-ui/src/mixins/resource-lookup-mixin.ts

```diff
@@ -59,8 +59,11 @@ export default class ResourceLookupMixin extends Vue {
       }
 
       if (configSection.canadianResident) {
-        if (directors.filter(x => (x.deliveryAddress.addressCountry !== 'CA' ||
-        (x.mailingAddress && x.mailingAddress.addressRegion !== 'BC'))).length === directors.length) {
+        const count = directors.length
+        const notCanadian = directors.filter(x => (x.deliveryAddress.addressCountry !== 'CA' ||
+        (x.mailingAddress && x.mailingAddress.addressCountry !== 'CA'))).length
```

Comment:
The requirements doc doesn't specify WHICH address(es) to use to determine whether a director had an address outside Canada. Is it where they live? Is it where they do business?

Please confirm this with a BA and then add a code comment indicating the decision so that reviewers/future devs know what the exact business rule is implemented here.

Reply:
It should be both mailing and delivery (I've been told). I will add comments explaining that.

Reply:
Both or either?

Reply:
None of their addresses should be outside of Canada

Reply:
Thanks for the comments in the code indicating the business rule that is implemented.